### PR TITLE
feat/text-equals

### PR DESCRIPTION
**Pull request by degrootsam**

### CODE REVIEW

**Comment by emiel-betty** [comment]
@degrootsam make sure to merge the latest `main` branch in yours.

File: __tests__/text-equals/text-equals.test.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+function returnExpectedResult(comparator) {
+  // Calculates the expected result for the given comparator
+  // This way the test can be automated
+  let expectedResult = false;
+  let left = leftValue;
+  let right = rightValue;
+
+  if (toLower) {
+    left = leftValue.toLowerCase();
+    right = rightValue.toLowerCase();
```
Comment (jwielinga) [inline __tests__/text-equals/text-equals.test.js:28]: Making the logic in the tests is not what you want. You want to test the outcome of the function that is being used

File: functions/equals-text/1.0/index.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+const equalsText = ({ leftValue, rightValue, comparator, toLower }) => {
+  console.log("🚀 ~ file: index.js:2 ~ equalsText ~ comparator", comparator)
```
Comment (jwielinga) [inline functions/equals-text/1.0/index.js:2]: Remove log, not really mandatory to log this for every exectution of the function right?

File: functions/equals-text/1.0/index.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  let result = false;
```
Comment (jwielinga) [inline functions/equals-text/1.0/index.js:3]: Don't declare the variable. Better practice to return from switch and have the defaulted value be `false`.

File: __tests__/text-equals/text-equals.test.js (hunk 1 above)
Comment (jwielinga) [reply __tests__/text-equals/text-equals.test.js:28]: So please refactor tests to a similar approach being used of the functions in the main branch

**Review by jwielinga — CHANGES_REQUESTED** [review]

File: functions/equals-text/1.0/index.js (hunk 3 above)
Comment (degrootsam) [reply functions/equals-text/1.0/index.js:3]: This is what I had first so I kindly refuse to go back again, not going to rebuild what was first denied.

**Pull request closed without merging**